com.microsoft.mrtk.input/Utilities/PoseSource/InputActionPoseSource.cs

```diff
@@ -33,9 +33,9 @@ public bool TryGetPose(out Pose pose)
             InputAction positionAction = positionActionProperty.action;
             InputAction rotationAction = rotationActionProperty.action;
 
-            if (trackingStateAction != null && trackingStateAction.enabled
-                && positionAction != null && positionAction.enabled
-                && rotationAction != null && rotationAction.enabled
+            if (trackingStateAction != null && trackingStateAction.controls.Count != 0
```

Comment:
Do we not care about the `enabled` state any more?

Reply:
Controls.count better captures whether the action is being used (we use this pattern in another spot in the codebase as well)